internal/webhook/config/config.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"strings"
+
+	"github.com/kelseyhightower/envconfig"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type WebhookConfig struct {
```

Comment:
Nit: idk about this needing its own package. Can we move it up one level? Or was this choice made to break a would-be import cycle?

Reply:
I followed how `api` server organize config, but no preferences over it. Will move under `webhook` :)